frontend/src/css/Sidebar.styl

```diff
@@ -170,11 +170,14 @@ sidebar-animate-speed = 0.5s
       position absolute
       min-height "calc(100% - %s)" % headerHeight
 
-@media (max-width: 575px)
+@media (max-width: max-xs)
   .sidebarpagecontainer
     position relative
 
   .sidebar
     &__frame
       position absolute
       min-height "calc(100% - %s)" % headerHeight
+
+  .iconbutton__icon
+    margin 0
```

Comment:
why on sidebar file ? shouldn't it be on icon button file ?

Reply:
you're right, I fixed this :)